lib/stat_tracker.rb

```diff
@@ -21,6 +21,60 @@ def initialize(files)
     end
   end
 
+  def highest_scoring_visitor 
+    #make a hash of of total away games by team id
+    away_games_by_team_id = @games.each_with_object(Hash.new(0.0)) do |game, hash|
+      hash[game.away_team_id] += 1
+    end
+    #make a hash of total away goals by team id
+    away_goals_by_team_id = @games.each_with_object(Hash.new(0.0)) do |game, hash|
+      hash[game.away_team_id] += game.away_goals
+    end
+    
+    #average them together, very similar to the average we did with #average_goals_by_season
+    average_away_goals_by_team_id = Hash.new(0.0)
+    away_goals_by_team_id.each do |key, value|
+      average_away_goals_by_team_id[key] = (value/ away_games_by_team_id[key]).round(2)
+    end
+    average_away_goals_by_team_id
```

Comment:
Wondering if this is an accidental variable? Not sure what Line 39 does

Reply:
yeah it doesn't do anything. I actually noticed after I pushed, but at easy fix!